src/main/js/dev/rest/localisation/translations_sv.json

```diff
@@ -402,11 +344,12 @@
   "registryItem.deleteCancel": "Ta inte bort anordnare",
   "registryItem.deleteConfirm": "Bekräfta radering",
   "registryItem.extra": "Ytterligare information",
-  "registryItem.extra.placeholder": "Till exempel Allmän e-postlista alla@anordnare",
+  "registryItem.extra.placeholder": "Till exempel Allmän e-postlista alla@anordnare.fi",
   "registryItem.merchantId": "Handlarens ID",
   "registryItem.merchantSecret": "Handlarens lösenord",
   "registryItem.merchantSecret.mandatory": "Obligatorisk om handlarens ID har angetts",
   "registryItem.payment": "Betalningstjänstens uppgifter",
   "registryItem.paymentInfoMissing": "Betalningstjänstens uppgifter fattas",
-  "registryItem.search.placeholder": "Sök med organisationens namn"
-}
+  "registryItem.search.placeholder": "Sök med organisationens namn",
+  "registryItem.viewAsOrganiser": "Visa anordnarens vy"
```

Comment:
This translation is missing from untuva. Also I take it's from Tarja or Saara?

Reply:
Itse käännetty, näkyy vain virkailijoille.